AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -206,49 +218,37 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
     }
 
     get isCourtroomAccount() {
-        return !this.emailValidationService.hasCourtroomAccountPattern(this.judgeName.value, this.invalidPattern);
+        return this.judge.is_courtroom_account;
     }
 
-    public addJudge(judgeEmail: string) {
-        if (!judgeEmail) {
-            return;
-        }
-        const selectedJudge = this.availableJudges.find(j => j.email === judgeEmail);
-        if (!selectedJudge) {
-            return;
-        }
-        this.judge.first_name = selectedJudge.first_name;
-        this.judge.last_name = selectedJudge.last_name;
-        this.judge.email = selectedJudge.email;
-        this.courtAccountJudgeEmail = selectedJudge.email;
-        if (!this.isJudgeDisplayNameSet()) {
-            this.judge.display_name = selectedJudge.display_name;
+    get displayEmailField() {
+        if (this.judge != null && this.judge.is_courtroom_account) {
+            return true;
+        } else {
+            return false;
         }
-        this.judgeDisplayNameFld.setValue(this.judge.display_name);
-        const newJudge = AssignJudgeComponent.mapJudgeToModel(this.judge);
-        this.hearing.participants = this.hearing.participants.filter(x => !x.is_judge);
-        this.hearing.participants.unshift(newJudge);
-        this.hearing = Object.assign({}, this.hearing);
     }
 
     isJudgeDisplayNameSet(): boolean {
         let result = false;
         if (this.judge && this.judge.display_name) {
-            const enteredJudge = this.availableJudges.find(j => j.display_name === this.judge.display_name);
-            result = !enteredJudge;
+            result = true;
```

Comment:
I think you could one line this whole func now that it's simpler

Reply:
Agreed, thanks